content/_guides/helpers.md

```diff
@@ -0,0 +1,165 @@
+asking, answer it and ignore the typos.
+
+### Crossposting
+
+It may feel like a waste to answer someone asking the same question in
+multiple channels. Sometimes people don't have much time, or maybe they have
+a boss hurrying them! Seeking second opinions is also a perfectly normal
+social practice; people do it all the time.
+
+Confronting people about occasional crossposting makes the channel feel
+hostile, and people feel like they are under surveillance, regardless of why
+they do it. If someone wants to answer the question, let them. If you don't
+want to answer, ignore it quietly.
+
+## They are being rude
+
+While it would be very nice if people were never rude, it **is** going to
+happen. However,
+[don't automatically assume people are rude](catalyst#assume-good-faith).
+They may have different cultural norms or a language barrier that
+unintentionally changes their perceived tone.
+
+It is best to let small things slide and not raise the channel temperature by
+calling them out. If a person's demeanor bothers you, quietly refrain from
+engaging with them instead of telling them off. If a channel member is
+hostile, it is important not to worsen the situation by responding in kind.
+
+## They are in the wrong place
+
+While you may know how to navigate IRC, other people might be confused or not
+know where to begin. Instead of scolding people for getting the wrong channel,
+it is better to say you do not know the answer and tell them the correct
+channel to ask.
+
+When you redirect people, first check if they can join the destination
+channel. Otherwise, it might seem like you're pushing them away. You may need
+to `/whois` them to find out if they are identified or not:
+
+```irc
+/whois Nickname
+```
+
+If they are identified, there will be a line saying so. It looks like this:
+
+```irc
+[Nickname] is logged in as Accountname
+```
+
+If they are not identified, check if they will unable to join because of `+r`
+in the channel modes, or if they will be unable to talk because of a `+q` on
+`$~a` (users that are not identified). This is how you would check for the
+`#windows` channel:
+
+```irc
+/mode #windows
+/mode #windows q
+```
+
+At the time of writing, `#windows` does not have either of these restrictions
+set, but if it did, the output of the above might look like:
+
+```irc
+11:49 -- Mode #windows [+Ccnptjfr]
+11:49 -- Channel #windows created on Wed, 19 May 2021 07:36:22
+11:49 -- [#windows] $~a quieted by copper.libera.chat on Mon, 14 Jun 2021 10:49:19
+11:49 -- [#windows] End of Channel Quiet List
+```
+
+If the user needs to be identified, and they are not, they might need
+reminding. You can do this when redirecting them politely.
+
+```irc
+<You> I don't know much about Windows, but the people in #windows will! You will need to be registered to talk in there. If you have an account, see '/msg nickserv help identify'. If not, see '/msg nickserv help register' :)
```

Comment:
I wonder if we should be consistent with how atheme docs present commands here, and use same capitalisation in the example. Also would it be nicer to use a channel name for a FOSS/PDP project rather than windows?


```suggestion
<You> I don't know much about Windows, but the people in #windows will! You will need to be registered to talk in there. If you have an account, see '/msg NickServ HELP IDENTIFY'. If not, see '/msg NickServ HELP REGISTER' :)
```